README.md

```diff
@@ -144,6 +146,51 @@ A `RequirementType` of `All` changes the traversal. First, if there are no filte
 
 In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning all of its filters must evaluate to true for the feature to be enabled. In this case, the feature will be enabled for 50% of users during the specified time window.
 
+#### Microsoft Feature Flag Schema
+
+The feature management library also supports the usage of [`Microsoft Feature Flag schema`](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json) to declare feature flags.
+
+If you want to use the `Microsoft Feature Flag schema`, you will need to follow the Azure App Configuration Feature Management Configuration schema to set up feature flags. The schema can be found [here](https://github.com/Azure/AppConfiguration/tree/main/docs/FeatureManagement). The feature flag declarations should be listed in the `FeatureFlags` array under the `FeatureManagement` section, as shown in the example below:
+
+```json

```

Comment:
I would reduce the complexity of this to a single feature flag. with a single filter. I think the biggest gain of the example is the fact that there is a `FeatureFlags` property and it is an `array`. Other than that we can include that single flag to demonstrate that the flag indeed uses the microsoft feature flag schema rather than the .net feature management schema.